testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-all/jvm.config

```diff
@@ -0,0 +1,19 @@
+-server
```

Comment:
Why do you add this file?

Reply:
The Ignite driver accessed the module ``java.nio`` module, it's need care while running on JDK17, so added ``--add-opens=java.base/java.nio=ALL-UNNAMED`` here.

Reply:
Having entire file is problematic. Consider a case of updating the version of ignite in the future. One will have issue to find out what has changed? Is it possible to apply a diff or configure env variables or something else to avoid changing the Ignite product code?